### PR TITLE
Fixes issue #3 : (added save feature)

### LINKED ISSUE #3: Implement feature to save canvas to device storage
Implement a feature to save canvas to device storage, so we can assess it later.

### PR DESCRIPTION
**Pull request by RockingRishav**
Fix #3
Screenshot_20221006-224657_undraw
Screenshot_20221006-224740_Gallery
Screenshot_20221006-224746_Gallery

### CODE REVIEW

**Comment by RockingRishav** [comment]
@rishidyno sir, could you please review my PR and give suggestion if required otherwise merge it with hacktoberfest-accepted label.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -20,6 +21,16 @@
...
+            android:exported="false"
+            android:grantUriPermissions="true"
+            >
+            <meta-data
+                android:name = "android.support.FILE_PROVIDER_PATHS"
+                android:resource="@xml/path"/>
+        </provider>
     </application>
 
 </manifest>
```
Comment (rishidyno) [inline app/src/main/AndroidManifest.xml]: EOL is missing here

File: app/src/main/java/com/rihsi/dyno/undraw/MainActivity.kt [hunk 2]
```diff
@@ -139,6 +160,62 @@ class MainActivity : AppCompatActivity() {
...
+                }
+                catch(e: Exception){
+                    toSave = ""
+                    e.printStackTrace()
+                }
+            }
+        }
+        return toSave
+    }
 }
```
Comment (rishidyno) [inline app/src/main/java/com/rihsi/dyno/undraw/MainActivity.kt:191]: remove extra line

File: app/src/main/res/xml/path.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+   <external-path
+       name="captured"
+       path="Android/media/com.rihsi.dyno.undraw/files"/>
+</paths>
```
Comment (rishidyno) [inline app/src/main/res/xml/path.xml]: Add EOL

**Review by rishidyno — CHANGES_REQUESTED** [review]
Add a screenshot showing the saved images in the device storage.

**Comment by rishidyno** [comment]
Please also delete 2 generated files.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (RockingRishav) [reply app/src/main/AndroidManifest.xml]: sir what does EOL stands for?

**Comment by rishidyno** [comment]
@RockingRishav I have updated the wiki PTAL look there and do the required changes.

**Comment by RockingRishav** [comment]
@rishidyno sir, i have applied the changes please recheck once

**Comment by rishidyno** [comment]
> @rishidyno sir, i have applied the changes please recheck once

I want you to locate it in phone.

**Comment by RockingRishav** [comment]
@rishidyno sir, Finally implemented the save to gallery feature please review it.

**Comment by rishidyno** [comment]
LGTM and Merged.

**Pull request merged**